### ISSUE TITLE
Average Response Time Graph blanks

### ISSUE THREAD

**Issue by jaxxstorm** [keen/pingpong#59]
I've deployed to heroku in a standard manner, and I can see data in my keen collection, however my average response time graph is completely blank.

Anything I can do to debug?

**Comment by josephwegner** [comment]
Hey @jaxxstorm - are you seeing any errors in your web console, or in your server logs?

**Comment by jaxxstorm** [comment]
nope, no errors at all. I can see the checks running successfull and the keen data exists, but no graphs :(

**Comment by wanghwh** [comment]
I have same issue.  I installed a standalone pingpong on ubuntu. the check is running , but if click into the check, no graph is shown

**Comment by hex337** [comment]
Are there any javascript errors on the page that loads the graphs?
